1-Authentication/1-sign-in/README.md

```diff
@@ -0,0 +1,256 @@
+ 1. [Setup](#setup)
+ 1. [Registration](#registration)
+ 1. [Running the sample](#running-the-sample)
+ 1. [Explore the sample](#explore-the-sample)
+ 1. [About the code](#about-the-code)
+ 1. [More information](#more-information)
+ 1. [Community Help and Support](#community-help-and-support)
+ 1. [Contributing](#contributing)
+
+## Overview
+
+This sample demonstrates an Angular single-page application (SPA) that lets users sign-in with Azure Active Directory (Azure AD) using the [Microsoft Authentication Library for Angular (Preview)](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-angular) (MSAL Angular). In doing so, it also illustrates various authentication concepts, such as [ID Tokens](https://docs.microsoft.com/azure/active-directory/develop/id-tokens), [OIDC scopes](https://docs.microsoft.com/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes), [single-sign on](https://docs.microsoft.com/azure/active-directory/develop/msal-js-sso), **silent requests** and more.
+
+## Scenario
+
+1. The client Angular SPA uses **MSAL Angular** to obtain an ID Token from **Azure AD**.
+2. The **ID Token** proves that the user has successfully authenticated against **Azure AD**.
+
+![Overview](./ReadmeFiles/topology.png)
+
+## Contents
+
+| File/folder                     | Description                                               |
+|---------------------------------|-----------------------------------------------------------|
+| `AppCreationScripts/`           | Contains Powershell scripts to automate app registration. |
+| `ReadmeFiles/`                  | Contains illustrations and etc.                           |
+| `src/app/auth-config.ts`        | Authentication parameters reside here.                    |
+| `src/app/app.module.ts`         | MSAL Angular configuration parameters reside here.        |
+| `src/app/app-routing.module.ts` | Configure your MSAL-Guard here.                           |
+
+## Prerequisites
+
+- An **Azure AD** tenant. For more information see: [How to get an Azure AD tenant](https://docs.microsoft.com/azure/active-directory/develop/quickstart-create-new-tenant)
+- A user account in your **Azure AD** tenant. This sample will not work with a **personal Microsoft account**. Therefore, if you signed in to the [Azure portal](https://portal.azure.com) with a personal account and have never created a user account in your directory before, you need to do that now.
+
+## Setup
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```console
+    git clone https://github.com/Azure-Samples/ms-identity-javascript-angular-tutorial.git
+```
+
+or download and extract the repository .zip file.
+
+> :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+
+### Step 2: Install project dependencies
+
+```console
+    cd ms-identity-javascript-angular-tutorial
```

Comment:
>ms-identity-javascript-angular-tutorial [](start = 7, length = 39)

the git clone does not create this folder, so I'd suggest something like cd <cloned folder>